exercises/intro_to_plugins/config.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "Hello Plugin",
+    "description": "Module that provides a freindly environment to admins and users alike",
```

```suggestion
    "description": "Module that provides a friendly page to announce its own existence",
```
Fix typo, remove language indicating user permissions